RHEL/6/input/remediations/bash/service_rpcbind_disabled.sh

```diff
@@ -0,0 +1,10 @@
+# platform = Red Hat Enterprise Linux 6
+#
+# Disable rpcbind for all run levels
+#
+/sbin/chkconfig --level 0123456 rpcbind off
+
+#
+# Stop rpcbind if currently running
+#
+/sbin/service rpcbind stop
```

Comment:
Please update this remediation script to look like the following:
https://github.com/OpenSCAP/scap-security-guide/blob/master/RHEL/6/input/remediations/bash/service_ypbind_disabled.sh